build_files/scripts/branding.sh

```
@@ -0,0 +1,76 @@
+chmod "$original_permissions" /usr/lib/os-release
+
+check_and_modify() {
+    local file="$1"
+    local pattern="$2"
+    local replacement="$3"
+    
+    if [ -f "$file" ]; then
+		original_permissions=$(stat -c "%a" "$file")
+        
+		# Create temporary file 
+		local temp_file=$(mktemp)
```

Shellcheck

[warning] Declare and assign separately to avoid masking return values.

(SC2155)